src/components/modal/CdrModal.vue

```diff
@@ -190,6 +201,62 @@ const removeNoScroll = () => {
   }
 };
 
+const backgroundContentNodes: Array<HTMLElement> = [];
+let backgroundNodesDiscovered = false;
+
+/**
+ * findBackgroundContentNodes
+ * When modal opens for the first time, find the content that should
+ * be hidden when it opens, and store it in an array
+ */
+const findBackgroundContentNodes = () => {
+  // initial selector is complex, breaking it down
+  const notSelectors = [
+    'body > *',
+    ':not(script)',
+    ':not(style)',
+    ':not([data-is-modal])',
+    ':not([aria-hidden=true])',
+  ]
+  const contentBodyChildren: NodeListOf<HTMLElement> =
+    document.querySelectorAll(notSelectors.join(''));
+
+  // for the remaining, check if they should be hidden
+  contentBodyChildren.forEach((el) => {
+    if (getComputedStyle(el).getPropertyValue('display') !== 'none') {
+      // TODO - temp to be sure we tagged right
+      el.setAttribute('data-modal-hide', '');
+      backgroundContentNodes.push(el);
+    }
+  });
+  backgroundNodesDiscovered = true;
+}
+
+/**
+ * hideModalBackgroundContent
+ * Hide elements tagged onBeforeMount from screen readers
+ */
+const hideModalBackgroundContent = () => {
+  // only run the expensive DOM scraping once
+  if (!backgroundNodesDiscovered) {
+    findBackgroundContentNodes();
```

Comment:
Put this here instead of in a `beforeMount` hook, for example.  
Two reasons:

1. Performance: this may never need to run at all.
2. Our pages continue loading content at the body level way past when the `DOMContentLoaded` event fires.  Playing around with this in the PDP, I had to use a setTimeout of something like 5 seconds (on a fast connection) to get all of our little `body` children loaded.